### PR TITLE
Add capella and elixir branches to antora-playbook.yml for Staging

### PR DESCRIPTION
**Pull request by sarahlwelton**

### CODE REVIEW

File: workspace/search/search-capella
Comment (osfameron) [inline workspace/search/search-capella:1]: did you mean to add these workspaces as submodules?

**Review by osfameron — CHANGES_REQUESTED** [review]
Oh, yeah, this is docs-site, we don't want those workspace submodules

File: workspace/search/search-capella
Comment (sarahlwelton) [reply workspace/search/search-capella:1]: Oh whoops let me fix that.

**Review by simon-dew — CHANGES_REQUESTED** [review]
Just one addition please.

(I need to keep the Data API reference material in cb-swagger for the moment, which means it needs to have capella and elixir branches for now &mdash; which means it needs to move here. This may be just a temporary solution.)

File: .gitignore [hunk 1]
```diff
@@ -13,5 +13,6 @@ xref-validator.log
 **/package-lock.json
 **/lab.sh
 **/.DS_Store
+/workspace
```
Comment (osfameron) [inline .gitignore:16]: 👍

**Review by osfameron — APPROVED** [review]
thanks 👍

**Review by simon-dew — APPROVED** [review]

**Pull request merged**